engine/src/main/java/org/camunda/bpm/engine/impl/calendar/DurationHelper.java

```diff
@@ -80,6 +80,7 @@ public DurationHelper(String expressions, Date startDate) throws Exception {
       start = DateTimeUtil.parseDate(expression.get(0));
       if (isDuration(expression.get(1))) {
         period = parsePeriod(expression.get(1));
+        times = Math.max(0, times - 1);
```

Comment:
Maybe a code comment would be nice here, to clarify why we reduce by 1, so devs in the future can understand it more quickly.

Reply:
Good hint. I added a comment.